app/flows/check_travel_during_coronavirus_flow/questions/which_country.erb

```diff
@@ -1,9 +1,9 @@
 <% text_for :title do %>
   <% calculator ||= nil %>
   <% if calculator && calculator.countries.count >= 1 %>
-    Which other countries will you be going to?
+    Which other country will you be going to?
   <% else %>
-    Which countries will you be going to?
+    Which country will you be going to?
   <% end %>
```

Comment:
You're right - this reads in an odd way. I feel we should change this to be simply...

```ruby
<% text_for :title do %>
  <% calculator ||= nil %>
  Which country will you be going to?
<% end %>
``` 

At least then the header, and the `Your answers` text will all be consistent. As you say we'll replace this soon enough anyway.